src/_data/side-nav.yml

```diff
@@ -82,10 +58,21 @@
           match-page-url-exactly: true
           permalink: /null-safety/faq
 
-    - title: Evolution
-      permalink: /guides/language/evolution
-    - title: Specification
-      permalink: /guides/language/spec
```

Comment:
**question:** I can understand these being resources, but with the heading as given, why not keep these here?

Reply:
The intention is that progressing down the list of topics under the "language" umbrella teaches readers the Dart language. The evolution and specification pages are extra resources, and:
1. aren't necessary to learn Dart
2. don't match the type of content under language
3. would detract from the visual message we're trying to convey (here's everything you need to know about the Dart language)